consensus/hotstuff_handler.go

```diff
@@ -46,3 +46,10 @@ func (m *consensusModule) shouldElectNextLeader() bool {
 	// Execute leader election if there is no leader and we are in a new round
 	return m.step == NewRound && m.leaderId == nil
 }
+
+func (m *consensusModule) shouldHandleHotstuffMessage(step typesCons.HotstuffStep) bool {
```

Comment:
I pulled the branch locally and searched for `shouldHandleHotstuffMessage` but couldn't find any places where this is being used.

Reply:
Yes it seems it is not being used anywhere, removed.